app/views/mail/validated.scala.html

```diff
@@ -1,10 +1,14 @@
 @( name: String, uri: String )
-Hello @name,
 
-your account was activated. You can start tracing at
+@emailBaseTemplate() {
+  <p>Hi @name,</p>
 
-@uri
+  <p>
+    Congratulations, your account is now activated and you are ready to start exploring and to collaboratively
```

Comment:
Congratulations sounds like a bit too much to me, what about:

```suggestion
    Great news, your account is now activated and you are ready to start exploring and to collaboratively
```